### PR TITLE
chore: flesh out Rust SDK page

### PR DESCRIPTION
**Pull request by cprice404**

### CODE REVIEW

**Comment by cprice404** [comment]
Still need to poke around in the Rust SDK itself and add the appropriate snippets.

**Comment by cprice404** [comment]
depends on this PR https://github.com/momentohq/client-sdk-rust/pull/317

File: docs/sdks/rust/cache.mdx [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+```
+export MOMENTO_API_KEY=<your Momento API key here>
+```
+
+## Import libraries and instantiate a CacheClient object
+
+This code sets up the main function, the necessary imports, and the CacheClient instantiation that each of the other functions will need to call.
+
+<SdkExampleCodeBlock language={'rust'} file={'./example/src/bin/cheat_sheet.rs'} />
```
Comment (anitarua) [inline docs/sdks/rust/cache.mdx]: missing cheat sheet? or should this be `cache.rs`?

Comment (cprice404) [reply docs/sdks/rust/cache.mdx]: see this PR: https://github.com/momentohq/client-sdk-rust/pull/317

Comment (cprice404) [reply docs/sdks/rust/cache.mdx]: oops i might have somehow missed adding the file to that PR, double-checking.

**Review by anitarua — APPROVED** [review]

**Pull request merged**